relay.go

```diff
@@ -17,8 +24,85 @@ type RelayHosts interface {
 }
 
 type relayItem struct {
+	*time.Timer
+
 	remapID     uint32
 	destination *Relayer
+	tomb        bool
+}
+
+type relayItems struct {
+	sync.RWMutex
+
+	logger Logger
+	tombs  uint64
+	items  map[uint32]relayItem
+}
+
+func newRelayItems(logger Logger) *relayItems {
+	return &relayItems{
+		items: make(map[uint32]relayItem),
+	}
+}
+
+// Count returns the number of items in the relay.
+func (r *relayItems) Count() int {
+	r.RLock()
+	n := len(r.items)
+	r.RUnlock()
+	return n
+}
+
+// Get checks for a relay item by ID. It returns the item, if it's still active,
+// whether the item is a tombstone,
```

Comment:
comment seems incorrect (and truncated) -- `Get` just returns whether something was found.